validate.sh

```diff
@@ -369,7 +368,6 @@ step_time_summary() {
 step_build() {
 print_header "build"
 timed $CABALNEWBUILD $TARGETS --dry-run || exit 1
-$CABALPLAN topo --builddir=$BUILDDIR || exit 1
```

Comment:
If we still want a list of packages (though not topologically sorted) we can do it with a jq command:

```sh
jq -r '."install-plan" | map(."pkg-name" + "-" + ."pkg-version" + " " + ."component-name") | join("\n")' "$BUILDDIR/cache/plan.json"
```

Reply:
TBH I don't see a big value in this list. If someone feels strongly about it, I could add this oneliner, perhaps. But I'm not a fan of this solution either: it looks rather fragile to my uniformed taste.

Reply:
Ah, I guess, it was requested here: https://github.com/haskell/cabal/issues/8440 so there are users for it...

Reply:
>  One difference between cabal build --dry-run and cabal-plan topo that I noticed today (and you can check it using the CI link above) is that the latter seems to print nearly twice as much as the former. I'm not sure why that is...

`cabal build` does not print cached stuff

Reply:
Good to know, thank you!

I don't see much emotion one way or another, so I'm just leaving the current version (with your oneliner) for the review...

Reply:
Where is the  one-liner replacement for the `cabal-plan` printing?  I cannot see it, I only see the deletion of the `cabal-plan` step.

Reply:
@andreasabel oops: forgot to commit. Fixed.